test/controllers/api/v2/bookmark_test.rb

```diff
@@ -0,0 +1,25 @@
+# encoding: utf-8
+
+require "katello_test_helper"
+
+module Foreman
+  class Api::V2::BookmarksControllerTest < ActionController::TestCase
+    test_attributes :pid => '6e7e4e9b-f12d-4e88-9d21-6c2f858e142b'
+    def test_create_with_valid_katello_controller
```

Comment:
Testing Foreman features with Katello is great, but could we test that the bookmark works too?

We've definitely seen the in the past where Foreman happily accepts a name from a plugin's namespace but the code doesn't actually work.